src/huntsman/pocs/states/huntsman/coarse_focusing.py

```diff
@@ -1,7 +1,6 @@
-from panoptes.utils.time import wait_for_events, current_time
-
-from panoptes.utils import altaz_to_radec
-from panoptes.pocs.utils.location import create_location_from_config
+from panoptes.utils.time import CountdownTimer
+from panoptes.utils.time import current_time
```

Comment:
```suggestion
from panoptes.utils.time import CountdownTimer, current_time
```

Reply:
Unfortunately we don't have a set style policy...